editor/editor_file_system.cpp

```diff
@@ -68,6 +68,10 @@ int EditorFileSystemDirectory::find_dir_index(const String &p_dir) const {
 	return -1;
 }
 
+void EditorFileSystemDirectory::force_update() {
+	modified_time = 0;
```

Comment:
Might need an explanatory comment, it's not obvious how changing the `modified_time` will force an update.

Reply:
`// We set modified_time to 0 to force `EditorFileSystem::_scan_fs_changes` to search changes in the directory`

seems good?

Reply:
Yeah :)